workflow-templates/DataMiner-CICD-UpdateCatalogDetails.yml

```diff
@@ -0,0 +1,19 @@
+name: DataMiner CICD Update Catalog Details
+
+# Controls when the workflow will run
+on:
```

Comment:
Either extend with already configured paths for the Readme & generated catalog yml, or provide the link so that the developer can choose: 
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore